.github/workflows/docker-pr.yaml

```diff
@@ -21,42 +21,45 @@ jobs:
 
       - name: Linting
         run: |
-          hadolint scripts/tvm_cli/Dockerfile
-          hadolint scripts/tvm_cli/Dockerfile.dependencies.amd64
-          hadolint scripts/tvm_cli/Dockerfile.dependencies.arm64
+          hadolint scripts/tvm_cli/Dockerfile.amd64
+          hadolint scripts/tvm_cli/Dockerfile.arm64
 
   build-docker:
       needs: dockerfile-lint
 
-      runs-on: ubuntu-latest
+      runs-on: ${{ matrix.os }}
+      strategy:
+        matrix:
+          os: [ubuntu-latest, ARM64]
```

Comment:
You'll either need to install Docker as part of this script or run tthis in a different container. The default container used on our `ARM64` runner does not have Docker installed so it can't run the build script. I recommend trying `docker:dind` (Docker in Docker). Make sure you set it at the `job` level. Same goes for all jobs in all scripts in this repo that are for building Docker images.